src/containers/parameters/index.jsx

```diff
@@ -338,7 +117,20 @@ function ParamNetwork({ location }) {
         justifyContent="center"
         flexDirection="column"
       >
-        {content}
+        {param === 'bandwidth' && <BandwidthParam data={dataParam.bandwidth} />}
+        {param === 'slashing' && <SlashingParam data={dataParam.slashing} />}
+        {param === 'staking' && <StakingParam data={dataParam.staking} />}
```

Comment:
maybe here would be better map from config

Reply:
up to you)